syft/event/parsers/parsers.go

```diff
@@ -109,17 +110,17 @@ func ParseFileIndexingStarted(e partybus.Event) (string, progress.StagedProgress
 	return path, prog, nil
 }
 
-func ParsePresenterReady(e partybus.Event) (presenter.Presenter, error) {
+func ParsePresenterReady(e partybus.Event) (*formats.SBOMWriter, error) {
```

Comment:
I also am a bit confused about go pointers vs structs. I will do a little reading, but could use a bit of explanation when to use one vs the other...